bootstrap-app/src/cmr/bootstrap/data/metadata_retrieval/collection_metadata_cache.clj

```diff
@@ -0,0 +1,123 @@
+(ns cmr.bootstrap.data.metadata-retrieval.collection-metadata-cache
+  "Defines a cache for catalog item metadata. It currently only stores collections.
+  The metadata cache contains data like the following:
+
+  concept-id -> revision-format-map
+                 * concept-id
+                 * revision-id
+                 * native-format - A key or map of format and version identifying the native format
+                 * various format keys each mapped to compressed metadata."
+  (:require
+   [clj-time.core :as t]
+   [clojure.set :as cset]
+   [cmr.common-app.data.metadata-retrieval.collection-metadata-cache :as cmn-coll-metadata-cache]
+   [cmr.common-app.data.metadata-retrieval.revision-format-map :as crfm]
+   [cmr.bootstrap.data.metadata-retrieval.metadata-transformer :as metadata-transformer]
+   [cmr.common.config :refer [defconfig]]
+   [cmr.common.hash-cache :as hash-cache]
+   [cmr.common.jobs :refer [defjob]]
+   [cmr.common.log :as log :refer [info]]
+   [cmr.common.util :as u]
```

Comment:
too short, u vs util

Reply:
Yeah, I copied that code from the original, I will change it.